scoring_engine/checks/ldap.py

```diff
@@ -2,16 +2,15 @@
 
 
 class LDAPCheck(BasicCheck):
-    required_properties = ['domain', 'base_dn']
-    CMD = "ldapsearch -x -h {0} -p {1} -D {2}@{3} -b {4} -w {5} '(objectclass=User)' cn"
+    required_properties = ['domain_base']
+    CMD = "ldapsearch -x -H 'ldap://{0}:{1}' -D 'cn={2},cn=Users,{3}' -w '{4}' -b '{3}' '(objectclass=Domain)' cn"
```

Comment:
ldapsearch now has different arguments in its current version which is why the change. Also was tweaked to change the objectClass to reduce check output size